### PR TITLE
CI: Benchmark tweaks

### PR DESCRIPTION
**Pull request by str4d**
Part of zcash/halo2#419.

### CODE REVIEW

File: benches/plonk.rs [hunk 1]
```diff
@@ -247,61 +244,74 @@ fn bench_with_k(name: &str, k: u32, c: &mut Criterion) {
...
+    }
+    keygen_group.finish();
+
+    let mut prover_group = c.benchmark_group("plonk-prover");
+    prover_group.sample_size(10);
+    for k in k_range.clone() {
+        let (params, pk) = keygen(k);
+
+        prover_group.bench_with_input(
+            BenchmarkId::from_parameter(k),
```
Comment (str4d) [inline benches/plonk.rs:292]: @ebfull this change will create benchmark names like `plonk-prover/11`, which assumes this benchmark will only vary `k`. Does that seem likely to change at all, or should I alter the naming to make it `plonk-prover/k/11`?

Comment (daira) [reply benches/plonk.rs:292]: This seems fine to me. It doesn't preclude adding other parameters later.

**Review by daira — APPROVED** [review]
utACK

File: benches/plonk.rs (hunk 1 above)
Comment (str4d) [reply benches/plonk.rs:292]: Yeah, we can always go into the `gh-pages` branch and rename the existing benchmarks to preserve past data.

**Pull request merged**